nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -70,7 +70,15 @@ in {
       '';
     };
 
-    nginx.enable = mkEnableOption "nginx vhost management";
+    nginx.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable nginx vhost management.
+        Further nginx configuration can be done by adapting <literal>nginx.virtualHosts.&lt;name&gt;</literal>.
+        See <link xlink:href="https://nixos.org/nixos/options.html#nginx.virtualhosts.%3Cname%3E"/> for further information.
```

Comment:
@grahamc do you know how one can refer to manual options instead?

Reply:
> 
> 
> this can be achieved with `<xref linked="<id-of-chapter-to-link>" />`

How can I find the ID of chapter ?

Reply:
I found and correct on  #48041